core/settings.py

```diff
@@ -93,6 +90,7 @@
     CAS_SERVER_URL = env("CAS_SERVER_URL")
     CAS_VERSION = env("CAS_VERSION")
     INSTALLED_APPS += ("django_cas_ng",)
+    MIDDLEWARE += "django_cas_ng.middleware.CASMiddleware",
```

Comment:
This is working because of the trailing comma but I strongly suggest
```suggestion
    MIDDLEWARE += ["django_cas_ng.middleware.CASMiddleware"]
```